source/fundamentals/data-formats/document-data-format-bson.txt

```diff
@@ -13,11 +13,82 @@ Document Data Format: BSON
 Overview
 --------
 
-- Definition of BSON format, where it's used, why it's used.
-- Link to more info on BSON
-- What information you can find in the following sections
+This guide explains the BSON data format, how MongoDB uses it, and how to
+install the BSON library independently of the MongoDB driver.
 
-Installation
-------------
+BSON Data Format
+----------------
 
-- Instructions on how to add the dependency with Maven or Gradle
+**BSON**, or Binary JSON, is the data format that MongoDB uses to organize
+and store data. This data format includes all JSON data structure types and
+adds support for binary types. BSON is a serialized form of JSON optimized for 
```

Comment:
and other types, e.g. dates, different size integers, ObjectId, etc.

Reply:
Maybe link to https://docs.mongodb.com/manual/reference/bson-types/?

Reply:
Ok, will include those types. I did link to bson-types in the next sentence:
```
:manual:`BSON Types </reference/bson-types>`
```

Reply:
Added the types info.